.vscode/settings.json

```diff
@@ -1,5 +0,0 @@
-{
-    "editor.formatOnSave": true,
```

Comment:
Feel like this should not have been committed to code. The tab width and quote styles are already covered via Prettier configuration.